src/SpellsReference/Components/Spell/Create.js

```diff
@@ -70,15 +84,15 @@ class Create extends Component {
             </div>
             <div className="col-sm-2">
               <label htmlFor="schoolOfMagic">School</label>
-              <select id="schoolOfMagic" className="form-control" defaultValue="abjuration">
-                <option value="0">Abjuration</option>
-                <option value="1">Conjuration</option>
-                <option value="2">Divination</option>
-                <option value="3">Enchantment</option>
-                <option value="4">Evocation</option>
-                <option value="5">Illusion</option>
-                <option value="6">Necromancy</option>
-                <option value="7">Transmutation</option>
+              <select id="schoolOfMagic" className="form-control" defaultValue="0">
+                <option>Abjuration</option>
+                <option>Conjuration</option>
+                <option>Divination</option>
+                <option>Enchantment</option>
+                <option>Evocation</option>
+                <option>Illusion</option>
+                <option>Necromancy</option>
+                <option>Transmutation</option>
```

Comment:
Just a note:
Web API 2 is actually pretty good about parsing strings to enums with its model binding, so I leveraged that instead of using integers.